lib/src/main/java/uk/gov/di/ipv/core/library/domain/gpg45/validation/Gpg45EvidenceValidator.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.di.ipv.core.library.domain.gpg45.validation;
+
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.core.library.domain.gpg45.Gpg45Profile;
+import uk.gov.di.ipv.core.library.domain.gpg45.domain.CredentialEvidenceItem;
+
+public class Gpg45EvidenceValidator {
+    @ExcludeFromGeneratedCoverageReport
+    private Gpg45EvidenceValidator() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    public static boolean validate(CredentialEvidenceItem item, Gpg45Profile gpg45Profile) {
+
+        if (item.getStrengthScore() < gpg45Profile.scores.evidences().get(0).strength()) {
+            return false;
+        }
+        if (item.getValidityScore() < gpg45Profile.scores.evidences().get(0).validity()) {
+            return false;
+        }
+        return item.getCi() == null || item.getCi().isEmpty();
+    }
```

Comment:
It looks like the `validate` method on these validators is no longer being used. The CIs are being checked separately, and the scores are being compared directly to the profiles. They could probably be removed, and a bunch of tests with them.

Reply:
Ah yeah good shout. Thanks I'll remove these and the tests